### ISSUE TITLE
HiDPI issues persist on Linux

### ISSUE THREAD

**Issue by zebra-ok** [gephi/gephi#2423]
## Expected Behavior
Icons are recognizable, buttons are clickable, input fields are visible and it's possible to type something in it, ...

## Current Behavior
Icons are very small, buttons especially with only 1 icon are very small, some inputs are not accessible, sliders are not large enough, statistics report diagrams are not scaled

## Steps to Reproduce

1. Use Gephi on Linux with a monitor with high density pixel per inch 
2. Observe scaling issues

## Context

My monitor has a 4k resolution and a global scaling factor of 200%

## Your Environment

* Version used: Gephi 0.9.3 nightly with latest commits built myself on 30.01.22
* Java version: openjdk 11.0.14
* Operating System: Fedora Linux

![Screenshot2](https://user-images.githubusercontent.com/11131859/152690343-d4c126c6-29e9-4f44-9551-020c8cab0df1.png)
![Screenshot1
](https://user-images.githubusercontent.com/11131859/152690384-13315707-b2c1-412c-8f14-8528b496fb74.png)

![Screenshot3](https://user-images.githubusercontent.com/11131859/152690390-09ccf5f4-0006-4a1d-b2c3-f5c3e82255b0.png)

**Comment by krempelra** [comment]
Hi,
I had this problem too. I never found a solution but it seems its a Netbeans thing.
Have you tried things like these:
https://stackoverflow.com/a/70872981

**Comment by krempelra** [comment]
Netbeans seems to have fixed a lot of issues for Highres in the latest version of Netbeans, the dependency  in gephi is up to date. But they havent fixed all issues concering bugs with highres scaling.
https://cwiki.apache.org/confluence/display/NETBEANS/HiDPI+%28Retina%29+improvements
But i cant really say if thats one of the things they haven't fixed. 
Have you tried other Netbeans applications?

**Comment by zebra-ok** [comment]
`-J-Dsun.java2d.uiScale=2` indeed works, thanks! Before, I only tried this parameter with the Gephi 0.9.2 release, and it didn't work there, but since I built `master` with JRE 11, it works. Still some minor bugs, but I will close this issue.

**Comment by mbastian** [comment]
Thanks, from a high level with the JRE11, Netbeans update and our own tweaks I feel we're okayish to release the 0.9.3 version as is on Linux. HiDPI seems to work fine but some problem remains when scaling is activated. We'll pick it up for the future 0.9.4 version.

**Comment by krempelra** [comment]
Just to explain the in between Fix . I used the latest 0.9.3 Snapshot Version. 

i added the parameter `-J-Dsun.java2d.uiScale=2`  to the `default_options="` Variable in  _etc/gephi.conf_ .

Like
`default_options="--branding gephi -J-Dsun.java2d.metal=true -J-Dsun.java2d.noddraw=true `...
 becomes:
`default_options="--branding gephi -J-Dsun.java2d.metal=true -J-Dsun.java2d.uiScale=2 -J-Dsun.java2d.noddraw=true `...

Im using Ubuntu 20.4 LTS with a Highres Screen and 200% scaling.

**Comment by mbastian** [comment]
Thanks a lot. I'll try it out locally and see if that's something we could fix without needing an extra parameter.

**Comment by mbastian** [comment]
I tried this `-J-Dsun.java2d.uiScale=2` extra parameter on Ubuntu/Gephi 0.9.3 with a 200% scaling and it **does indeed work perfectly**. 

I tried to understand why this isn't automatically picked up by Java and/or Netbeans and it does seem like an open issue. That should hopefully be fixed at some point. I don't think we should try to automate that ourselves. Netbeans has [this open issue](https://issues.apache.org/jira/browse/NETBEANS-2360).
